### PR TITLE
b-fix/gh-1208-log-in-issue-after-dropping-the-database

### PR DESCRIPTION
**Pull request by popovicbojan**
Resolves: #1208

### CODE REVIEW

File: src/main/java/eu/execom/hawaii/security/CustomUserDetailsService.java [hunk 1]
```diff
@@ -36,4 +36,11 @@ public User loadUserById(Long id) {
                          .orElseThrow(
                              () -> new EntityNotFoundException("User with given id doesn't exist in database."));
   }
+
+  @Transactional
```
Comment (stanislavgrujic) [inline src/main/java/eu/execom/hawaii/security/CustomUserDetailsService.java]: Why do you need this to be transactional? There is only a read command over the database.

File: src/main/java/eu/execom/hawaii/security/UserPrincipal.java [hunk 2]
```diff
@@ -37,6 +37,8 @@ Long getId() {
     return id;
   }
 
+  String getEmail() { return email; }
```
Comment (stanislavgrujic) [inline src/main/java/eu/execom/hawaii/security/UserPrincipal.java]: Please do not inline method body. Always try to keep the code style that is used throughout the codebase which you joined.

File: src/main/java/eu/execom/hawaii/security/CustomUserDetailsService.java (hunk 1 above)
Comment (popovicbojan) [reply src/main/java/eu/execom/hawaii/security/CustomUserDetailsService.java]: I saw that methods loadUserByUsername() and loadUserById() above are all transactional, so I thought that all similar methods needed to be transactional, although it doesn't make a sense. Should I remove transactional annotation from all of them?

File: src/main/java/eu/execom/hawaii/security/UserPrincipal.java (hunk 2 above)
Comment (popovicbojan) [reply src/main/java/eu/execom/hawaii/security/UserPrincipal.java]: Ok, I'll correct this.

File: src/main/java/eu/execom/hawaii/security/CustomUserDetailsService.java (hunk 1 above)
Comment (stanislavgrujic) [reply src/main/java/eu/execom/hawaii/security/CustomUserDetailsService.java]: Please do. :)

**Pull request merged**